libs/gltfio/src/AssetLoader.cpp

```diff
@@ -564,6 +567,8 @@ void FAssetLoader::recurseEntities(const cgltf_data* srcAsset, const cgltf_node*
         float3* scale = (float3*) &node->scale[0];
         float3* translation = (float3*) &node->translation[0];
         localTransform = composeMatrix(*translation, *rotation, *scale);
```

Comment:
we should remove localTransform here, and use the new Trs transform manager to calculate it, so that it's done the same way everywhere.

Reply:
FIXED